sw/device/silicon_creator/lib/crc32.c

```diff
@@ -34,14 +36,36 @@ void crc32_add8(uint32_t *ctx, uint8_t byte) {
   }
 }
 
+void crc32_add32(uint32_t *ctx, uint32_t word) {
```

Comment:
nit: should the context be a typedef'ed struct so that clients of the crc32 library have an opaque interface?

Reply:
I thought about that but it seemed obvious that crc32 is 32-bits and felt like overkill. I can definitely define a type if you feel strongly about it.

Reply:
Keeping this as `uint32_t`.